packages/react-ui-ag/src/Listings/__tests__/SingleFamilyDesktopListing-test.js

```diff
@@ -0,0 +1,331 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import { omit } from 'lodash'
+import { Button, ToggleButton, ListingComponents } from '@rentpath/react-ui-core'
+import ThemedSingleFamilyDesktopListing from '../Listing'
+import theme from './mocks/theme'
+
+const SingleFamilyDesktopListing = ThemedSingleFamilyDesktopListing.WrappedComponent
+
+const baseListing = {
+  banners: ['New Construction', 'Sponsored'],
+  bedrooms: '1-3 Beds',
+  bathrooms: '2 Baths',
+  unitLevelAvailability: '12 Units Available',
+  availability: 'Available Now',
+  name: 'Awesome Property!',
+  address: '3921 Tugaloo River Drive Northside Park Washington Parkway',
+  city: 'Great Town',
+  state: 'YA',
+  price: '$1170+',
+  phone: '678-907-1428',
+  rating: {
+    score: 4,
+    label: '20',
+    fillColor: 'yellow',
+    backgroundFillColor: '#ffffff',
+    uniqueId: '123',
+  },
+  photos: [
+    {
+      path: '',
+      caption: '',
+    },
+    {
+      path: '',
+      caption: '',
+    },
+    {
+      path: '',
+      caption: '',
+    },
+  ],
+}
+
+const props = {
+  listing: baseListing,
+  theme,
+  onClick: () => { },
+  navigation: {
+    next: {
+      children: 'Next',
+    },
+    previous: {
+      children: 'Previous',
+    },
+  },
+  ctaButtons: [
+    {
+      children: 'Check Availability',
+      onClick: () => { },
+      className: 'contact',
+    },
+    {
+      children: '$ Coupon',
+      onClick: () => { },
+      className: 'coupon',
+      'data-tid': 'coupon-button',
+      'data-tag_item': 'coupon',
+    },
+  ],
+  favoriteButton: {
+    onClick: () => { },
+    children: '♥',
+  },
+  photos: {
+    server: 'https://image.rent.com/',
+    dimensions: '280-120',
+  },
+  lazyLoad: false,
+  listingInfoComponents: () => {
+    const listing = baseListing
+    const { phone } = listing
+
+    // return (
```

Comment:
Could you remove this commented out code?

Reply:
I forgot to remove that. Thanks for the catch! :)